README.md

```diff
@@ -15,3 +15,16 @@ Interface paths:
 /decode-data - Tags 'sophia-type' and 'data'.
 
 /encode-calldata - Tags 'source', 'function' and 'arguments'.
+
+/version - return the version of the Sophia compiler
+
+/api-version - return the version of the API
+
+/api - return the API in a JSON-term (intended to be consumed by tools)
+
+## Building
+
+The most convenient way to start the an HTTP server serving the Sophia compiler API is using
+docker. `make docker` will create a docker image `aeternity/aesophia_http:latest` and it is
```

Comment:
this is not true - it creates `aeternity/aesophia_http:local` as mentioned in the next row